### ISSUE TITLE
npn errors

### ISSUE THREAD

**Issue by SteveRMann** [OpenZWave/node-openzwave-shared#237]
Tried to follow the instructions, but two problems came up.  First, "Go to your project directory,
cd /<project directory>".  What project directory?

Second, the instruction line: "npm node-openzwave-shared" is wrong, I suspect you mean "npm install node-openzwave-shared".  But when I do, I get the following error:

> Not found : node-openzwave-shared
> npm ERR! 404  'node-openzwave-shared' is not in the npm registry.
> npm ERR! 404 You should bug the author to publish it...

So, I am stuck at step three.

**Comment by ekarak** [comment]
the NPM package name is 'openzwave-shared', apologies for the confusion. 
BTW which instructions are you following? this typo was fixed some time ago.

**Comment by SteveRMann** [comment]
Thanks for the prompt reply..
I am using the instructions in:
https://raspberrypihq.com/how-to-create-a-z-wave-smart-home-hub-using-a-raspberry-pi/
Which looks like a copy of the instructions in this git:
https://github.com/OpenZWave/node-openzwave-shared/blob/master/README-raspbian.md
And the confusing Step three is corrected.

But, in Step three, what is meant by "project directory"?

Thanks again.

**Comment by SteveRMann** [comment]
Makes sense.  I am trying to use Node Red to talk to some ZWave devices, using the Pi that is also my MQTT broker as the ZWave hub using the Aeotek ZStick.

So, the "last" step says to install _npm install node-red-contrib-openzwave_ in ~/.node-red.
With that environment, where would you put _node-openzwave-shared_?

I have no plans to write any other code for ZWave, I just want to use Node Red in my home IOT.

**Comment by ekarak** [comment]
you should install openzwave-shared under .node-red then.

**Comment by SteveRMann** [comment]
Many thanks, that's exactly the info I've been looking for.

**Comment by SteveRMann** [comment]
Thanks for the prompt reply..
I am using the instructions in:
https://raspberrypihq.com/how-to-create-a-z-wave-smart-home-hub-using-a-raspberry-pi/

**Comment by SteveRMann** [comment]
I give up.
Copied a fresh SD (from a backup of my working system) and followed the instructions in the git, and step three:
 _cd ~/.node-red
 npm install openzwave-shared_
You can test node-openzwave-shared
 _wget https://raw.githubusercontent.com/OpenZWave/node-openzwave-shared/master/test2.js
 node test2.js_
still fails with tons of "... cannot find module..." errors.
